develop/addons/schema-driven-forms/customising-form-behaviour/validation.rst

```diff
@@ -129,7 +129,105 @@ For example:
     ...
```

Comment:
Make this a comment so that it renders correctly with syntax highlighting. The Python Pygments lexer fails with invalid syntax.

```suggestion
    # ...
```